posts/2024/2024_06_07_Inigo_week_1.rst

```diff
@@ -0,0 +1,29 @@
+First Week into GSoC 2024: Building the AutoEncoder, preliminary results
+========================================================================
+
+.. post:: June 7 2024
```

Comment:
There is still confusion in dates and weeks. If I'm not mistaken there was a blog post (the first one) that corresponded to the community bonding period (week prior to May 27), and since then there have been two weeks: week 1 (ended Fri May 31), and week 2 (ended Fri Jun 7). So I find that this blog post still confuses dates and contents. Can this please be addressed? 2 blog posts are requested. To avoid this in the future, best is to allocate 30-45 minutes every Friday to write about the work that week. I know it is the most scientifically interesting part for the contributor, but it is important for the mentors, to keep track of the project, and the program.

Reply:
You are right, I messed up the dates. I will split the info into two blogs. Sorry for the confusion.